sqrl-tools/sqrl-cli/src/test/resources/snapshots/com/datasqrl/cmd/TestCmd/compileJsonDowncasting.txt

```diff
@@ -61,7 +63,7 @@ CREATE TEMPORARY TABLE `orders$2` (
   `id` BIGINT NOT NULL,
   `customerid` BIGINT NOT NULL,
   `time` VARCHAR(2147483647) CHARACTER SET `UTF-16LE` NOT NULL,
-  `entries` ROW(`productid` INTEGER NOT NULL, `quantity` INTEGER NOT NULL, `unit_price` DOUBLE NOT NULL, `discount` DOUBLE) NOT NULL ARRAY NOT NULL,
+  `entries` RAW('class com.datasqrl.json.FlinkJsonType', 'AEdvcmcuYXBhY2hlLmZsaW5rLmFwaS5qYXZhLnR5cGV1dGlscy5ydW50aW1lLmtyeW8uS3J5b1NlcmlhbGl6ZXJTbmFwc2hvdAAAAAIAH2NvbS5kYXRhc3FybC5qc29uLkZsaW5rSnNvblR5cGUAAATyxpo9cAAAAAIAH2NvbS5kYXRhc3FybC5qc29uLkZsaW5rSnNvblR5cGUBAAAAIQAfY29tLmRhdGFzcXJsLmpzb24uRmxpbmtKc29uVHlwZQEAAAAlAB9jb20uZGF0YXNxcmwuanNvbi5GbGlua0pzb25UeXBlAAAAAAApb3JnLmFwYWNoZS5hdnJvLmdlbmVyaWMuR2VuZXJpY0RhdGEkQXJyYXkBAAAAKwApb3JnLmFwYWNoZS5hdnJvLmdlbmVyaWMuR2VuZXJpY0RhdGEkQXJyYXkBAAAB8AApb3JnLmFwYWNoZS5hdnJvLmdlbmVyaWMuR2VuZXJpY0RhdGEkQXJyYXkAAAACrO0ABXNyAEJvcmcuYXBhY2hlLmZsaW5rLmFwaS5jb21tb24uRXhlY3V0aW9uQ29uZmlnJFNlcmlhbGl6YWJsZVNlcmlhbGl6ZXJBDr5NoiCPtQIAAUwACnNlcmlhbGl6ZXJ0ACZMY29tL2Vzb3Rlcmljc29mdHdhcmUva3J5by9TZXJpYWxpemVyO3hwc3IAbW9yZy5hcGFjaGUuZmxpbmsuYXBpLmphdmEudHlwZXV0aWxzLnJ1bnRpbWUua3J5by5TZXJpYWxpemVycyRTcGVjaWZpY0luc3RhbmNlQ29sbGVjdGlvblNlcmlhbGl6ZXJGb3JBcnJheUxpc3QAAAAAAAAAAQIAAHhyAGFvcmcuYXBhY2hlLmZsaW5rLmFwaS5qYXZhLnR5cGV1dGlscy5ydW50aW1lLmtyeW8uU2VyaWFsaXplcnMkU3BlY2lmaWNJbnN0YW5jZUNvbGxlY3Rpb25TZXJpYWxpemVyAAAAAAAAAAECAAFMAAR0eXBldAARTGphdmEvbGFuZy9DbGFzczt4cHZyABNqYXZhLnV0aWwuQXJyYXlMaXN0eIHSHZnHYZ0DAAFJAARzaXpleHAAAATyxpo9cAAAAAAAAATyxpo9cAAAAAA='),
```

Comment:
This strategy may not be viable. Flink seems to have some trouble recovering this class, I'll have to dig in more.

Reply:
'class com.datasqrl.json.FlinkJsonType' should have been 'com.datasqrl.json.FlinkJsonType'